doc/rm/c_cpp_coding_style.md

```diff
@@ -171,6 +171,67 @@ While this does correctly set the ABI for a header implemented in C, that header
 
 This last rule is waived for third-party headers, which may be polyglot but not declared in our style.
 
+### X Macros
+
+In order to avoid repetitive definitions or statements, we allow the use of [X Macros](https://en.wikipedia.org/wiki/X_Macro) in our C and C++ code.
+
+Uses of X Macros should follow the following example, which uses one in a switch definition:
+```c
```

Comment:
What does `c` mean here?

Reply:
it means highlight the code as if it were C.